cmd/adapter/adapter.go

```diff
@@ -120,6 +124,10 @@ func (cmd *PrometheusAdapter) addFlags() {
 		"kubeconfig file used to configure auth when connecting to Prometheus.")
 	cmd.Flags().StringVar(&cmd.PrometheusCAFile, "prometheus-ca-file", cmd.PrometheusCAFile,
 		"Optional CA file to use when connecting with Prometheus")
+	cmd.Flags().StringVar(&cmd.PrometheusTLSCertFile, "prometheus-tls-cert-file", cmd.PrometheusTLSCertFile,
+		"Optional TLS cert file to use when connecting with Prometheus")
+	cmd.Flags().StringVar(&cmd.PrometheusTLSKeyFile, "prometheus-tls-key-file", cmd.PrometheusTLSKeyFile,
+		"Optional TLS key file to use when connecting with Prometheus")
```

Comment:
nit: let's document that key/certificate rotation is not handled here and must be implemented elsewhere, i.e. a sidecar.

Reply:
another nit: as this refers to client certificates, can we call this `prometheus-client-tls-cert-file` and `prometheus-client-tls-key-file`?

Reply:
Changed